SasquatchTV/SasquatchTVObjC/AppDelegate.m

```diff
@@ -58,30 +57,33 @@ - (void)crashes {
   [MSCrashes setUserConfirmationHandler:(^(NSArray<MSErrorReport *> *errorReports) {
 
                // Use MSAlertViewController to show a dialog to the user where they can choose if they want to provide a crash report.
-               MSAlertController *alertController = [MSAlertController alertControllerWithTitle:@"Sorry about that!"
+               UIAlertController *alertController = [UIAlertController alertControllerWithTitle:@"Sorry about that!"
                                                                                         message:@"Do you want to send an anonymous crash "
-                                                                                                @"report so we can fix the issue?"];
+                                                                                                @"report so we can fix the issue?"
+                                                                                 preferredStyle:UIAlertControllerStyleAlert];
 
                // Add a "No"-Button and call the notifyWithUserConfirmation-callback with MSUserConfirmationDontSend.
-               [alertController addCancelActionWithTitle:@"Don't Send"
-                                                 handler:^(UIAlertAction *action) {
-                                                   [MSCrashes notifyWithUserConfirmation:MSUserConfirmationDontSend];
-                                                 }];
+               [alertController addAction:[UIAlertAction actionWithTitle:@"Don't Send"
```

Comment:
Run formatter - `:`s should align, there shouldn't be 2 parameters on the same line.  Hopefully the formatter will fix it automatically 🤷‍♂

Reply:
@jwallra it's not 2 parameters of the same function.

Reply:
I also ran a script for formatting.